cse-valid-adapter-app/src/main/java/com/farao_community/farao/cse_valid/adapter/app/service/CseValidAdapterService.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse_valid.adapter.app.service;
+
+import com.farao_community.farao.cse_valid.adapter.app.configuration.CseValidAdapterConfiguration;
+import com.farao_community.farao.cse_valid.adapter.app.exception.CseValidAdapterServiceException;
+import com.farao_community.farao.cse_valid.api.resource.CseValidFileResource;
+import com.farao_community.farao.cse_valid.api.resource.CseValidRequest;
+import com.farao_community.farao.gridcapa.task_manager.api.ProcessFileDto;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskDto;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskStatus;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskStatusUpdate;
+import com.farao_community.farao.gridcapa_cse_valid.starter.CseValidClient;
+import org.apache.commons.lang3.NotImplementedException;
+import org.slf4j.Logger;
+import org.slf4j.MDC;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.stereotype.Service;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+/**
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
+@Service
+public class CseValidAdapterService {
+
+    private static final String TTC_ADJUSTMENT_FILE_TYPE = "TTC_ADJUSTMENT";
+    private static final String CRAC_FILE_TYPE = "CRAC";
+    private static final String CGM_FILE_TYPE = "CGM";
+    private static final String GLSK_FILE_TYPE = "GLSK";
+    private static final String TASK_STATUS_UPDATE = "task-status-update";
+
+    private final CseValidClient cseValidClient;
+    private final CseValidAdapterConfiguration cseValidAdapterConfiguration;
+    private final StreamBridge streamBridge;
+    private final Logger businessLogger;
```

Comment:
You have to define a Logger Bean so that the application can be launched. It can be done by defining a named Logger (see CseRunnerEventsLogger in gridcapa-cse or CseAdapterBusinessLogger in gridcapa-cse-adapter)